infer/tests/codetoanalyze/java/pulse/issues.exp

```diff
@@ -163,6 +163,8 @@ codetoanalyze/java/pulse/WriterLeaks.java, codetoanalyze.java.infer.WriterLeaks.
 codetoanalyze/java/pulse/WriterLeaks.java, codetoanalyze.java.infer.WriterLeaks.outputStreamWriterNotClosedAfterWriteBad():void, 8, PULSE_RESOURCE_LEAK, no_bucket, ERROR, [allocation part of the trace starts here,allocated by constructor java.io.OutputStreamWriter() here,memory becomes unreachable here]
 codetoanalyze/java/pulse/WriterLeaks.java, codetoanalyze.java.infer.WriterLeaks.fileWriterNotClosedAfterWriteBad():void, 8, PULSE_RESOURCE_LEAK, no_bucket, ERROR, [allocation part of the trace starts here,allocated by constructor java.io.FileWriter() here,memory becomes unreachable here]
 codetoanalyze/java/pulse/WriterLeaks.java, codetoanalyze.java.infer.WriterLeaks.pipedWriterNotClosedAfterConstructedWithReaderBad():void, 10, PULSE_RESOURCE_LEAK, no_bucket, ERROR, [allocation part of the trace starts here,allocated by constructor java.io.PipedWriter() here,memory becomes unreachable here]
+codetoanalyze/java/pulse/taint/Annotations.java, codetoanalyze.java.pulse.Annotations.markedSourceToMarkedSink():void, 1, TAINT_ERROR, no_bucket, ERROR, [source of the taint here: value returned from String Annotations.markedSource() with kind Simple,passed as argument #1 to void Annotations.markedSink(String) with kind Simple]
```

Comment:
For test cases we usually try to follow explicit naming conventions:
- "bad" suffix is for cases where Infer correctly reports an issue
- "good" suffix is for cases where Infer is not supposed to report anything
- "falsePositive" and "falseNegative" for respective misreporting cases

Reply:
Added `Bad` and `Ok` suffixes, no false positives in this one.